test/any/parsing/GeckoParser-test.js

```diff
@@ -40,4 +40,14 @@ describe('GeckoParser', function() {
     `);
     expect(p.value).toBeDefined();
   });
+  it('should report parse complete status to console', function() {
+    console.info = jasmine.createSpy();
+    var p = GeckoParser.create().logParse('interface Frobinator;', 'InterfaceFwdDecl');
+    expect(console.info).toHaveBeenCalledWith('Parse complete');
```

Comment:
Let's make the strings be constants on `BaseParser`. Then we can refer to the constants here.